src/lumigo_tracer/parsers/utils.py

```diff
@@ -196,10 +196,19 @@ def _is_supported_http_method(event: dict):
         and "headers" in event  # noqa
         and "requestContext" in event  # noqa
         and event.get("requestContext", {}).get("elb") is None  # noqa
+        or event.get("version", "") == "2.0"  # noqa
```

Comment:
can you add parenthesis?
It's confusing...

Reply:
👍